### ISSUE TITLE
Helpers package output breaks if in sub directory

### ISSUE THREAD

**Issue by randallmlough** [web-ridge/gqlgen-sqlboiler#30]
If I attempt to put the outputted helpers dir as a sub directory, for example `graph/helpers` then the helpers package name will be `package graph/helpers` which obviously doesn't work

```go

func main() {
	cfg, err := config.LoadConfigFromDefaultLocations()
	if err != nil {
		fmt.Fprintln(os.Stderr, "failed to load config", err.Error())
		os.Exit(2)
	}

	convertHelpersDir := "graph/helpers"
	sqlboilerDir := "models"
	gqlgenModelDir := "graphql_models"

	err = api.Generate(cfg,
		api.AddPlugin(gbgen.NewConvertPlugin(
			convertHelpersDir, // directory where convert.go, convert_input.go and preload.go should live
			sqlboilerDir,      // directory where sqlboiler files are put
			gqlgenModelDir,    // directory where gqlgen models live
		)),
      
       // ...
```

**Comment by RichardLindhout** [comment]
Thanks for your work. Let me know if this works in new release or on master

**Comment by RichardLindhout** [comment]
Fixed in v2.1.0. Thanks for your PR! 👌
